src/Picocrypt.go

```diff
@@ -515,7 +517,7 @@ func draw() {
 
 					giu.Row(
 						giu.Checkbox("Deniability", &deniability),
-						giu.Tooltip("Add plausible deniability to the volume"),
+						giu.Tooltip("Add plausible deniability to the volume\nIf enabled, the comments will be lost"),
```

Comment:
Some people may not use comments..so maybe better to say

\nIf enabled, comments will not work.

Or something like that